301-spinnaker-vmss-or-aks/azuredeploy.parameters.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "devopsDnsPrefix": {
+      "value": "GEN-UNIQUE"
+    },
+    "adminUsername": {
+      "value": "GEN-UNIQUE"
+    },
+    "sshPublicKey": {
+      "value": "GEN-SSH-PUB-KEY"
+    },
+    "adminPassword": {
+      "value": "GEN-PASSWORD"
+    },
+    "servicePrincipalAppId": {
+      "value": "GEN-GUID"
+    },
+    "servicePrincipalAppKey": {
+      "value": "GEN-PASSWORD"
+    },
+    "aksClusterName": {
+      "value": "GEN-UNIQUE"
+    },
+    "aksResourceGroup": {
+      "value": "GEN-UNIQUE"
+    }
+  }
+}
```

Comment:
@neil-yechenwei  There is a new line brought in. Please remove it

Reply:
> @neil-yechenwei There is a new line brought in. Please remove it

removed